### PR TITLE
Edit russia

### PR DESCRIPTION
**Pull request by JeanBaptisteScellier**
Hello,
I propose these updates on Russia's data:
- add exchange data and arrow between european Russia and Crimea
- edit some wrong direction of exchanges between Russia and some other states. It was assumed that the direction was from states based on their alphabetical order, which is not the case. I checked consistency on https://br.so-ups.ru/ to get the right one.
- add some exchange informations between Russia and Finland and Lituania even if it already existed with other sources but it can be useful
- changed the source for exchange between Russia and Georgia because the current one doesn't look like it is working, so I put the source to RU.fetch_exchange

### CODE REVIEW

**Comment by JeanBaptisteScellier** [comment]
I've added some french names for Russia and correct some mistakes in names about Ural and Crimea

File: config/exchanges.json [hunk 1]
```diff
@@ -1308,7 +1308,7 @@
       43.158267
     ],
     "parsers": {
-      "exchange": "GE.fetch_exchange"
+      "exchange": "RU.fetch_exchange"
```
Comment (madsnedergaard) [inline config/exchanges.json:1311]: Did you check that the `RU.py` parser returns this exchange data? :)

File: web/locales/fr.json [hunk 2]
```diff
@@ -1232,17 +1232,19 @@
...
         },
         "RU-2": {
             "countryName": "Russie",
             "zoneName": "Sibérie"
         },
         "RU-EU": {
-            "zoneName": "Russie"
+            "countryName": "Russie",
+            "zoneName": "Arctique"
         },
```
Comment (madsnedergaard) [inline web/locales/fr.json:1244]: Would you mind also doing this for the English language file? It's nice to have actual zones and not just the country name for all of them :)

**Review by madsnedergaard — COMMENTED** [review]
Nice work here! 💪 
A few minor comments

File: config/exchanges.json (hunk 1 above)
Comment (JeanBaptisteScellier) [reply config/exchanges.json:1311]: Yes I did!

File: web/locales/fr.json (hunk 2 above)
Comment (JeanBaptisteScellier) [reply web/locales/fr.json:1244]: No problem

**Review by madsnedergaard — APPROVED** [review]
Nice! 👏

**Pull request merged**